nn_surrogate_benchmark/model_evaluator.py

```diff
@@ -97,6 +99,7 @@ def evaluate_multiple_sets(
         train_loader: DataLoader | None = None,
         val_loader: DataLoader | None = None,
         test_loader: DataLoader | None = None,
+        grads = False,
```

Comment:
```
grads: bool = False
```

Reply:
I removed the variable completely as it was not used